mods/cnc/maps/nod07c/map.yaml

```diff
@@ -0,0 +1,772 @@
+MapFormat: 11
+
+RequiresMod: cnc
+
+Title: ORCA Theft (c)
```

Comment:
BTW, we're adding the mission numbers now in #11399, so you can already change this to `07c: Orca Theft`